src/common/services/Firebase/FirebaseConfig.tsx

```diff
@@ -1,16 +1,16 @@
 // Import the functions you need from the SDKs you need
 import { initializeApp } from 'firebase/app';
 
-import { getFunctions } from 'firebase/functions';
-import { getAuth } from 'firebase/auth';
-import { getFirestore } from 'firebase/firestore';
-import { getStorage } from 'firebase/storage';
+// import { getFunctions } from 'firebase/functions';
+// import { getAuth } from 'firebase/auth';
+// import { getFirestore } from 'firebase/firestore';
+// import { getStorage } from 'firebase/storage';
 
 // Emulators
-// import { getFunctions, connectFunctionsEmulator } from 'firebase/functions';
-// import { getAuth, connectAuthEmulator } from 'firebase/auth';
-// import { getFirestore, connectFirestoreEmulator } from 'firebase/firestore';
-// import { getStorage, connectStorageEmulator } from 'firebase/storage';
+import { getFunctions, connectFunctionsEmulator } from 'firebase/functions';
+import { getAuth, connectAuthEmulator } from 'firebase/auth';
+import { getFirestore, connectFirestoreEmulator } from 'firebase/firestore';
+import { getStorage, connectStorageEmulator } from 'firebase/storage';
 
```

Comment:
Can t deploy with emulator, please test your branch in dev environment, for prod it automatically deploy